vms/platformvm/block/builder/builder_test.go

```diff
@@ -100,7 +100,7 @@ func TestBuildBlockShouldReward(t *testing.T) {
 	}()
 
 	var (
-		now    = env.state.GetTimestamp()
+		now    = env.backend.Clk.Time()
```

Comment:
can't recall why I kept this. In this suite Durango is active.
I'd drop now and validatorStartTime altogether or just set validatorStartTime to chainTime, if we want to have an indication of what the actual start time will be before executing the tx